definitions.yaml

```diff
@@ -55,6 +55,19 @@ definitions:
               default:
                 description: The instance type used in new cluster, if not specified
                 type: string
+          docker_volume_size:
```

Comment:
I recommend highly to rename `docker_volume_size` to `docker_volume_size_gb`, in line with the `size_gb` attribute we have in memory and storage info on nodes. The `_gb` eliminates the guesswork regarding what unit is used. Assuming that we are using Gigabyte here.